client/signup/main.jsx

```diff
@@ -314,29 +326,53 @@ class Signup extends React.Component {
 	handleFlowComplete = ( dependencies, destination ) => {
 		debug( 'The flow is completed. Destination: %s', destination );
 
-		const isNewUser = !! ( dependencies && dependencies.username );
+		const { isNewishUser, existingSiteCount } = this.props;
+
+		const isNewUser = !! (
+			( dependencies && dependencies.username ) ||
+			( isNewishUser && dependencies && dependencies.siteSlug && 0 === existingSiteCount - 1 )
```

Comment:
I'm curious -- why `0 === existingSiteCount - 1` instead of `existingSiteCount === 1`?

Reply:
The count is "what's in signup dependencies now" - `1`.

i.e.., If `dependencies.siteSlug`, there is one site that is the _current_ site they have in the signup flow dependencies. So we need to consider that when looking at the counter.

Reply:
What I mean is -- the two expressions are equivalent (`0 === existingSiteCount - 1` is true if and only if `existingSiteCount` is equal to `1`), but I find the latter easier to read. Can we use it instead?

Reply:
I see. I'm actually thinking now that this could be slightly more future-proof if it was:

```
existingSiteCount - 1 <= 0
```

Reply:
Or, to your point:

```
existingSiteCount <= 1
```

Reply:
If that's still capturing the intention, then yes, that seems more robust. 👍

Reply:
Updated to `existingSiteCount <= 1`. Thanks again!